js/whisper-demo/utils.js

```diff
@@ -55,7 +55,7 @@ async function readResponse(response) {
 
 export function log(i) {
     console.log(i);
-    document.getElementById('status').innerText += `\n[${performance.now().toFixed(2)}] ` + i;
+    // document.getElementById('status').innerText += `\n[${performance.now().toFixed(2)}] ` + i;
```

Comment:
This function is designed for showing some status info on the page for users.

I suggest using `console.log()` instead for those info we just want to show in console.

Reply:
Fixed in https://github.com/Honry/onnxruntime-inference-examples/pull/1/commits/166eaf16c5b7526172e276ae09ec387bd6cc0aa4